agent/Makefile

```diff
@@ -10,11 +10,12 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-EXTRAGOARGS:=
+EXTRAGOARGS?="-v -count=1"
```

Comment:
Do we want to change this default everywhere?  It was set that way in CI to ensure that CI always ran all tests, but in normal development one might want to take advantage of Go's test result caching and skip tests where no code is modified.

Reply:
Yeah fair point, it would make more sense to just set this env var in the buildkite yaml file for now and default to nothing in the makefile, will update